firmware-binaries/Cargo.toml

```diff
@@ -13,8 +13,9 @@ codegen-units = 1
 
 [workspace]
 members = [
-  "fdt-read",
-  "hello",
+  "examples/fdt-read",
+  "examples/hello",
+  
   "processing-element-test",
```

Comment:
I didn't put it in examples because to me examples are just things that you could run but are not essential. This is used for the automated hardware in the loop tests, so it's more than just an example. Does that make sense?

Once we have the clock control or bootloader binaries those wouldn't be `example`s either